test/spec/unit/bootstraps/liveblogTest.js

```diff
@@ -140,7 +140,6 @@ define([
             it('sets up scroll event listener', function () {
                 liveblog.init();
 
-                expect(window.addEventListener).to.have.been.calledOnce;
```

Comment:
> Could we add creativeInjector to the mocked dependencies in this test spec\

Yes!

> You could then test that functions in the mock are called when expected

Not sure how to test something we don't actually do (replace the placeholder with some html and css) as I don't know how its supposed to look once it happens 😕